apis/hive/v1/gcp/machinepools.go

```diff
@@ -13,6 +13,18 @@ type MachinePool struct {
 	//
 	// +optional
 	OSDisk OSDisk `json:"osDisk"`
+
+	// NetworkProjectID specifies which project the network and subnets exist in when
+	// they are not in the main ProjectID.
+	// +optional
+	NetworkProjectID string `json:"networkProjectID,omitempty"`
+
+	// ServiceAccount is the email of a gcp service account to be used for shared
+	// vpn installations. The provided service account will be attached to nodes
```

Comment:
```suggestion
	// vpc installations. The provided service account will be attached to nodes
```

Reply:
I did notice this -- but copied from the upstream struct so I left it alone :shrug:

Reply:
No longer applicable -- ServiceAccount isn't part of the feature anymore.